src/provision_server.py

```diff
@@ -101,38 +98,43 @@ def print_conf(D, path):
 	# slurm.conf
 	C = parse_slurm_conf("{CPR}/conf/slurm.conf".format(CPR = shlex.quote(CLUST_PROV_ROOT)))
 	C[["ControlMachine", "ControlAddr", "AccountingStorageHost"]] = ctrl_hostname
-	C["SuspendExcNodes"] = ctrl_hostname + "-nfs"
+	del C["SuspendExcNodes"]
```

Comment:
We can just remove this from the slurm.conf template, so we don't have to delete the key here.

Reply:
Yes, sure. I will do that.